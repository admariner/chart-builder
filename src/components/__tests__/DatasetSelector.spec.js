import React from 'react'
import DatasetSelector from '../DatasetSelector'

it('renders loading', () => {
  snap(
    <DatasetSelector
      token="foo"
      defaultValue="test/bar"
      value=""
      limitToProjects
      onChange={() => {}}
    />
  )
})
