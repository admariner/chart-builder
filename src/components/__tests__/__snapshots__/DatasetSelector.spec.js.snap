// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loading 1`] = `
<select
  className="form-control"
  disabled={true}
  onChange={[Function]}
  value=""
>
  <option
    disabled={true}
    value=""
  >
    Loading...
  </option>
</select>
`;
